integration-cli/docker_cli_exec_test.go

```diff
@@ -316,19 +316,22 @@ func (s *DockerSuite) TestInspectExecID(c *check.C) {
 		c.Fatalf("ExecIDs should be empty, got: %s", out)
 	}
 
-	// Start an exec, have it block waiting for input so we can do some checking
-	cmd := exec.Command(dockerBinary, "exec", "-i", id, "sh", "-c", "read a")
-	execStdin, _ := cmd.StdinPipe()
-
+	// Start an exec, have it block so we can do some checking
+	cmd := exec.Command(dockerBinary, "exec", "-i", id, "sleep", "10")
```

Comment:
Anything that stays there and can be easily exited would work. Sending a control-c to quit top would be more code than using sleep that is guaranteed to exit.